components/Meta.jsx

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head';
+import PropTypes from 'prop-types';
+
+const SITE_TITLE = 'Build | Olas';
+const SITE_DESCRIPTION = 'Explore paths to build on Olas. Simplify your path to contributing and earning OLAS rewards.';
+const SITE_URL = 'https://build.olas.network/';
+const SITE_DEFAULT_IMAGE_URL = `${SITE_URL}/images/metatags-image.png`;
+
+const Meta = ({ title, description }) => {
+  const pageTitle = title ? `${title} | ${SITE_TITLE}` : SITE_TITLE;
+
+  return (
+    <Head>
+      <title>{pageTitle}</title>
+
+      <meta name="title" content={pageTitle || SITE_TITLE} />
```

Comment:
don't think this tag exists 🤔 
i don't imagine there's any harm if it doesn't though

Reply:
Sorry, did you mean the meta tag?

Reply:
yeah, correct, `<meta name="title" content={pageTitle || SITE_TITLE} />` 
meta tag with the name "title" isn't used, or parsed by search engines afaik
basically useless code, but harmless nonetheless